parser_library/src/workspaces/workspace_configuration.cpp

```diff
@@ -716,35 +763,20 @@ const program* workspace_configuration::get_program_normalized(
     return nullptr;
 }
 
-std::pair<parse_config_file_result, utils::resource::resource_location>
-workspace_configuration::try_loading_alternative_configuration(const utils::resource::resource_location& file_location)
-{
-    auto result = std::pair(parse_config_file_result::parsed,
-        utils::resource::resource_location::replace_filename(file_location, B4G_CONF_FILE));
-    auto& [parsed, configuration_url] = result;
-
-    if (auto it = m_b4g_config_cache.find(configuration_url); it == m_b4g_config_cache.end())
-        parsed = parse_b4g_config_file(configuration_url);
-    else if (!it->second.config.has_value()) // keep in sync with parse_b4g_config_file
-        parsed = it->second.diags.empty() ? parse_config_file_result::not_found : parse_config_file_result::error;
-
-    return result;
-}
-
 utils::resource::resource_location workspace_configuration::load_alternative_config_if_needed(
     const utils::resource::resource_location& file_location)
 {
-    const auto rl = file_location.lexically_normal();
-
-    if (get_program_normalized(rl))
```

Comment:
I think the idea here was that the pgm_conf.json should have priority.

Reply:
It seems that [this](https://github.com/BroadcomMFD-DevOps/che-che4z-lsp-for-hlasm/blob/4e4cae02189da7e1f10a8fe138ace0df12bca33e/parser_library/src/workspaces/workspace_configuration.cpp#L273) takes care of the preference (at least as to which libs to use).

But I think I'm still not seeing the bigger picture here => still looking into this

Reply:
`That` just select the root for relative paths.

If you have pgm_conf.json that mentions a file then the processor group specified in the .bridge.json was not used.